auditlog/docker-compose.yaml

```diff
@@ -0,0 +1,69 @@
+version: '2.1'
+services:
+  zookeeper:
+    image: debezium/zookeeper:${DEBEZIUM_VERSION}
+    ports:
+     - 2181:2181
+     - 2888:2888
+     - 3888:3888
+  kafka:
+    image: debezium/kafka:${DEBEZIUM_VERSION}
+    ports:
+     - 9092:9092
+    links:
+     - zookeeper
+    environment:
+     - ZOOKEEPER_CONNECT=zookeeper:2181
+     # For local development of auditlog-enricher
+     # - ADVERTISED_HOST_NAME=192.168.1.6
```

Comment:
ah obviously some changes while I reviewed. yes this should be commented out or set to the correct IP address of the host machine from the very beginning ;-)